src/uu/pr/src/pr.rs

```diff
@@ -0,0 +1,1343 @@
+#![crate_name = "uu_pr"]
+
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE file
+// that was distributed with this source code.
+//
+
+#[macro_use]
+extern crate quick_error;
+
+use chrono::offset::Local;
+use chrono::DateTime;
+use getopts::{HasArg, Occur};
+use getopts::{Matches, Options};
+use itertools::structs::KMergeBy;
+use itertools::{GroupBy, Itertools};
+use quick_error::ResultExt;
+use regex::Regex;
+use std::convert::From;
+use std::fs::{metadata, File, Metadata};
+use std::io::{stdin, stdout, BufRead, BufReader, Lines, Read, Stdin, Stdout, Write};
+use std::iter::{FlatMap, Map};
+use std::num::ParseIntError;
+#[cfg(unix)]
+use std::os::unix::fs::FileTypeExt;
+use std::vec::Vec;
+
+type IOError = std::io::Error;
+
+static NAME: &str = "pr";
```

Comment:
Could you please follow this example to declarations the options?
https://github.com/uutils/coreutils/blob/master/src/uu/ls/src/ls.rs#L78-L80